pkg/certificates/constants.go

```diff
@@ -17,8 +17,9 @@ limitations under the License.
 package certificates
 
 const (
-	Organization = "knative.dev"
-	FakeDnsName  = "data-plane." + Organization
+	Organization        = "knative.dev"
+	LegacyFakeDnsName   = "data-plane." + Organization
+	DataPlaneNamePrefix = "knative-"
```

Comment:
A lot of downstream repos seem to use the `FakeDnsName` constant - so this is breaking? Are you doing follow up PRs to the various net-* repos?

Reply:
I suggest we fix this here  - see https://github.com/knative-sandbox/control-protocol/pull/269
And later follow up with a more gradual change in dependencies.